docs/gno-tooling/cli/gnodev.md

```diff
@@ -0,0 +1,49 @@
+---
+id: gno-tooling-gnodev
+---
+
+# Gnodev
+
+Gnodev allows for quick and efficient development of Gno code.
+
+By watching your development directory, gnodev detects changes in your Gno
+code, reflecting them in the state of the node immediately. Gnodev also runs a
+local instance of `gnoweb`, allowing you to see the rendering of your Gno code instantly. 
+
+## Features
+- **In-Memory Node**: Gnodev starts an in-memory node, and automatically loads
+the **examples** folder and any user-specified paths.
+- **Web Interface Server**: Gnodev automatically starts a `gnoweb` server on
+[`localhost:8888`](https://localhost:8888).
+- **Hot Reload**: Gnodev monitors the **examples** folder and any specified for file changes,
+reloading and automatically restarting the node as needed.
+- **State Maintenance**: Gnodev replays all transactions in between reloads,
+ensuring the previous node state is preserved.
+
+## Installation
+Gnodev can be found in the `contribs` folder in the monorepo.
```

Comment:
Do we want to provide a relative path to the directory (`../../../contribs`)?

Reply:
I don't think so, because this will not render out properly on the docs website